eox_core/api/v1/serializers.py

```diff
@@ -216,6 +217,10 @@ def update(self, instance, validated_data):
         has_profile = hasattr(instance, 'profile')
 
         extra_registration_fields.update(extended_profile_fields)
+
+        # Initialize profile_meta to avoid potential unassigned variable warning
+        profile_meta = {}
+
```

Comment:
Can we remove line 235? Also, I'm guessing this is covered by unittests, right?

Reply:
Hi majo, I had to add that line since a test was failing with the message: E0606: Possibly using variable 'profile_meta' before assignment (possibly-used-before-assignment).